modules/home/assets/js/components/icons/youtube-icon.js

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
```

Comment:
It might be better to move the `/icons` folder outside of the `/compontents` folder. As they have done for the notifications app.

<img width="329" alt="Screenshot 2024-03-08 at 12 43 50" src="https://github.com/elementor/elementor/assets/94352322/dcce8f16-abf5-4a42-ba7c-522e802ea7bd">

Reply:
done